neurovault/apps/statmaps/views.py

```diff
@@ -350,6 +352,11 @@ def upload_folder(request, collection_cid):
                             request.FILES['file'] = f
                             populate_nidm_results(request,collection)
                             continue
+                        elif fnmatch(f.name,'*.feat.zip'):
```

Comment:
OK, that should be doable- the directory structure of the uploaded folder is lost, but it can be reconstructed with the `paths[]` post variable.

One thing that might need refinement, is a specification for conclusively detecting the directory types.  I can start with some reasonable assumptions for that, and I'm sure we can clarify it later if necessary.